package.json

```diff
@@ -10,6 +10,7 @@
     "test": "NODE_ICU_DATA=node_modules/full-icu skn test",
     "check": "lerna run check",
     "release": "yarn lint:changelogs && lerna version --no-push --include-merged-tags",
```

Comment:
The `yarn lint:changelogs` script makes sure you have the various version number headers in place in changelog files before lerna runs. This on longer needs to the the case now those changes are updated as part of running `lerna version`.

`yarn lint:changelogs` can be removed from here, and the whole script can be deleted is it is no longer relevant.